src/services/shims.ts

```diff
@@ -322,8 +323,19 @@ module ts {
         }
     }
 
+    function serializeResultAsJson(target: Shim, key: string | symbol, descriptor: PropertyDescriptor) {

```

Comment:
I guess there would have to be three, because one has to take the union, in case you are decorating a computed property that is a union of symbol and string.